packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/normal/services_ignore_none_exist_resource.yaml

```
@@ -0,0 +1,23 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+
+    App\Some\:
+        resource: '../src/NotExistingDir/*'
+-----
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+
+    $services->defaults()
+        ->autowire()
+        ->autoconfigure();
+
+    $services->load('    App\\Some\\', __DIR__.'/../src/NotExistingDir/*');
```

Is this typo or mall-produced by converter?

```suggestion
-    $services->load('    App\\Some\\', __DIR__.'/../src/NotExistingDir/*');
+    $services->load('App\\Some\\', __DIR__.'/../src/NotExistingDir/*');
```

---

Typo :)